Quick_Deploy/HuggingFaceTransformers/persimmon8b/config.pbtxt

```diff
@@ -0,0 +1,40 @@
+# Triton backend to use
+backend: "python"
+
+# Hugging face model path
+# Parameters must follow this
+# key/value structure
+parameters: {
+	key: "huggingface_model",
+	value: {string_value: "adept/persimmon-8b-base"}
+}
+
+# The maximum number of tokens
+# to generate in response to
+# our input
+parameters: {
+	key: "max_output_length",
+	value: {string_value: "15"}
+}
+
+# Triton should expect a single
+# string of set length named prompt
+# as input
+input [
+  {
+    name: "prompt"
```

Comment:
could we update these with "text-input" and "text-output" as the names to match up with vLLM backend? Not critical but nice to have.

Reply:
+1 on renaming

Also, since there's no limitations like optional inputs or decoupled mode being used here, we can define the I/O names in auto complete function instead of config file.

Reply:
The tutorial initially used auto-complete for the input and outputs, but in an [earlier review ](https://github.com/triton-inference-server/tutorials/pull/49#discussion_r1335027432)it was decided to add this information to the config.pbtxt files instead.

Reply:
Oops, OK let's keep it in config for tutorial for now then, sorry for the confusion.

Reply:
No worries!